backend/src/middleware/notifications/notificationsMiddleware.spec.js

```diff
@@ -154,18 +158,18 @@ describe('notifications', () => {
             await createCommentOnPostAction()
             const expected = expect.objectContaining({
               data: {
-                currentUser: {
-                  notifications: [
-                    {
-                      read: false,
-                      reason: 'comment_on_post',
-                      post: null,
-                      comment: {
-                        content: commentContent,
-                      },
+                notifications: [
+                  {
+                    read: false,
+                    createdAt: expect.any(String),
+                    reason: 'commented_on_post',
+                    from: {
+                      __typename: 'Comment',
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-30T08:08:49Z" title="Friday, August 30th 2019, 10:08:49 am +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -154,18 +155,16 @@ describe('notifications', () => {
             await createCommentOnPostAction()
             const expected = expect.objectContaining({
               data: {
-                currentUser: {
-                  notifications: [
-                    {
-                      read: false,
-                      reason: 'comment_on_post',
-                      post: null,
-                      comment: {
-                        content: commentContent,
-                      },
+                notifications: [
+                  {
+                    read: false,
+                    reason: 'comment_on_post',
+                    from: {
+                      __typename: 'Comment',
```


Cool as well !!! 😍
Love it …